src/components/ProductDetailInfo/ProductDetailInfo.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Typography from "material-ui/Typography";
+import Grid from "material-ui/Grid";
+
+/**
+ * Product detail basic info fields
+ * @class ProductDetailTitle
+ */
+class ProductDetailInfo extends Component {
+  static propTypes = {
+    /**
+     * Prodcut description
+     */
+    description: PropTypes.string,
+
+    /**
+     * Price or price range as a string
+     */
+    priceRange: PropTypes.string,
+
+    /**
+     * Product vendor
+     */
+    vendor: PropTypes.string
+  }
+  render() {
+    const { priceRange, vendor, description } = this.props;
+
+    // Render all props are undefined then skip rendering component
```

Comment:
I believe this is a typo, it should be something like: "If all props are undefined ..."